diff --git CMakeLists.txt CMakeLists.txt
index 07d0d9b..9864db4 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -7,7 +7,7 @@
 # Specify the required version of CMake.
 # cmake 3.10 required for ctest/gtest integration
 cmake_minimum_required(VERSION 3.10)
-project(ale VERSION 0.8.0 DESCRIPTION "Abstraction Library for Ephemerides ")
+project(ale VERSION 0.8.1 DESCRIPTION "Abstraction Library for Ephemerides ")
 
 # include what we need
 include(GNUInstallDirs)
@@ -39,7 +39,9 @@ if(ALE_BUILD_LOAD)
   # If there is an Anaconda environment activated, search that for Python first
   if(EXISTS $ENV{CONDA_PREFIX})
     message("Searching $ENV{CONDA_PREFIX} for Python libraries")
-    set(Python_ROOT_DIR $ENV{CONDA_PREFIX})
+    set(Python_FIND_VIRTUALENV FIRST)
+    set(Python_FIND_REGISTRY LAST)
+    set(Python_FIND_FRAMEWORK LAST)
     set(Python_FIND_STRATEGY LOCATION)
   endif()
   find_package(Python REQUIRED COMPONENTS Development)
@@ -59,6 +61,8 @@ set(ALE_HEADER_FILES ${ALE_BUILD_INCLUDE_DIR}/InterpUtils.h
                      ${ALE_BUILD_INCLUDE_DIR}/Orientations.h
                      ${ALE_BUILD_INCLUDE_DIR}/States.h
                      ${ALE_BUILD_INCLUDE_DIR}/Isd.h
+                     ${ALE_BUILD_INCLUDE_DIR}/Distortion.h
+                     ${ALE_BUILD_INCLUDE_DIR}/Vectors.h
                      ${ALE_BUILD_INCLUDE_DIR}/Util.h)
 set(ALE_PRIVATE_LINKS Eigen3::Eigen)
 set(ALE_PUBLIC_LINKS nlohmann_json::nlohmann_json)
