diff --git a/CMakeLists.txt b/CMakeLists.txt
index 07d0d9b..f821122 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -42,7 +42,7 @@ if(ALE_BUILD_LOAD)
     set(Python_ROOT_DIR $ENV{CONDA_PREFIX})
     set(Python_FIND_STRATEGY LOCATION)
   endif()
-  find_package(Python REQUIRED COMPONENTS Development)
+  find_package(Python 3.6 EXACT REQUIRED COMPONENTS Development)
 endif()
 
 # Library setup
