diff --git CMakeLists.txt CMakeLists.txt
index 62ed676..d462d55 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -17,7 +17,7 @@ set(CMAKE_CXX_STANDARD 11)
 # Third Party Dependencies
 find_package(GSL           REQUIRED)
 find_package(Eigen3 3.3    REQUIRED NO_MODULE)
-find_package(Python        REQUIRED COMPONENTS Development)
+find_package(Python 3.6 EXACT REQUIRED COMPONENTS Development)
 find_package(nlohmann_json REQUIRED)
 
 # Library setup
